### PR TITLE
Add pumpkin juice receipe

### PR DESCRIPTION
**Pull request by polgarj**

### CODE REVIEW

File: src/recipes/pumpkin-juice.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            "ingredient": "honey or maple syrup to taste (optional. I don't add it as I find the apple juice is sweet enough on its own)"
+        }
+    ],
+    "directions": [
+        "Combine all ingredients in a blender and pulse to combine.",
+        "Simmer for 5 minutes to help enhance the flavor of the spices infuse into the mixture.",
+        "Enjoy warm or cool and enjoy chilled.",
+        "Store leftovers in an air-tight jar or container in fridge for 3-4 days. Shake or mix well before serving after storing as the pumpkin will settle."
+    ],
+    "image": "aam-panna.jpg",
```
Comment (alfg) [inline src/recipes/pumpkin-juice.json:38]: Please provide an image for the recipe. Existing images within the repo cannot be used.

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/pumpkin-juice.json (hunk 1 above)
Comment (polgarj) [reply src/recipes/pumpkin-juice.json]: Sorry for the mistake. I just uploaded the image of the drink.

Comment (alfg) [reply src/recipes/pumpkin-juice.json]: Can you remove this file from the PR? 
`.history/src/recipes/pumpkin-juice_20211001204814.json`

Comment (polgarj) [reply src/recipes/pumpkin-juice.json]: I removed it. Sorry again. :)

Comment (alfg) [reply src/recipes/pumpkin-juice.json]: Hey, no worries! Thanks for the contribution!

**Pull request merged**